pkg/i18n/english.go

```diff
@@ -1746,6 +1759,9 @@ func EnglishTranslationSet() TranslationSet {
 			CopyCommitURLToClipboard:       "Copy commit URL to clipboard",
 			CopyCommitAuthorToClipboard:    "Copy commit author to clipboard",
 			CopyCommitAttributeToClipboard: "Copy to clipboard",
+			CopyBranchAttributeToClipboard: "Copy to clipboard",
```

Comment:
Just thinking out loud: I'm okay with having two of the attributes with the same text, but if it happens for a third time maybe it's best to make them all the same attribute.

Reply:
I think I'll just create a generic Copy to Clipboard attribute since the copy menu would have the same label everywhere.

Reply:
looks like there's already a CopyToClipboard attribute.